### PR TITLE
feature/semver-script

### PR DESCRIPTION
**Pull request by nathanmartinszup**
Signed-off-by: nathanmartinszup <nathan.martins@zup.com.br>

Adding semver script

### CODE REVIEW

**Comment by matheusalcantarazup** [comment]
Why do wee need this script?

**Review by lucasbrunozup — APPROVED** [review]

**Comment by wiliansilvazup** [comment]
> Why do wee need this script?

because, all actions created in community are working in progress, we can't usage this repositories, and so we create them to automatize release proccess in horusec.

**Review by wiliansilvazup — APPROVED** [review]

File: scripts/semver.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  case $Option in
+    M ) major=true;;
+    m ) minor=true;;
+    p ) patch=true;;
+  esac
+done
+
+shift $(($OPTIND - 1))
+
+#version=$1
```
Comment (matheusalcantarazup) [inline scripts/semver.sh:31]: This seems to be a dead code?

File: scripts/semver.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+echo "cd to github workspace"
+cd ${GITHUB_WORKSPACE}
+git for-each-ref refs/tags/ --count=1 --sort=-version:refname --format='%(refname:short)'
+
+version=$(git for-each-ref refs/tags/ --count=1 --sort=-version:refname --format='%(refname:short)')
```
Comment (matheusalcantarazup) [inline scripts/semver.sh:36]: Why execute twice the same command?

**Review by wiliansilvazup — APPROVED** [review]

**Pull request merged**